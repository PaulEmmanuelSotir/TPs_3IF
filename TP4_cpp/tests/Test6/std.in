EN_ERROR_MESS
S seg1 0 1 2 3
R rec 0 1 1 0
S seg2 3 1 2 0
LIST
EXIT
