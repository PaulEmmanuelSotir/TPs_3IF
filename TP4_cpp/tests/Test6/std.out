OK
OK
OK
OK
* seg1 :        { (0, 1); (2, 3) }
* rec : { (0, 1); (1, 0) }
* seg2 :        { (3, 1); (2, 0) }
